.github/workflows/issue_triage.yml

```diff
@@ -0,0 +1,17 @@
+name: Issue triage
+
+on:
+    issues:
+        types: ["labelled"]
+
+jobs:
+    assign_enhancement_issue:
+        name: Assign enhancement issue
+        runs-on: ubuntu-latest
+        if: ${{ github.event.label == "✨ enhancement" }}
```

Comment:
Tenkte i første omgang begrense dette til enhancement-issues (nye features, endringsforslag) siden bugs av og til er enklest at noen som kjenner koden godt tar tak i. Åpen for at vi også auto-assigner dem ;)

Reply:
Jeg synes det er en fin linje å legge seg på i utgangspunktet, så kan vi vurdere om vi skal automatisere self-assign for bugs senere. Jeg kan jo se for meg at det er lettere å ansvarliggjøre fremgang på innmeldte bugs _(enten gjennom egne kode-bidrag eller oppfølging)_ som har ligget lenge hvis det ligger et navn knyttet til issuet, uavhengig av om man kan fikse det selv eller ei 😄